portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
+  * Create a new file in the `/portafly` root folder, and name it `.env`
+  * Paste `REACT_APP_API_HOST="https://multitenant-admin.preview01.3scale.net"` into it, and Save
+3. Create an Access token:
+  * Access your local Porta build (or access [Preview](https://multitenant-admin.preview01.3scale.net/))
+  * Navigate to Settings > Personal > Tokens
+  * Click the "Add Access Token" button
+  * Give your access token a Name
+  * Select all scopes
+  * Set permissions to Read & Write
+  * Create the access token
+  * Copy the access token
+4. Allow CORS requests in your browser:
```

Comment:
We may explain that allowing CORS is only needed in development mode, and add a disclaimer that if you decide to add an extension to bypass CORS, is at your own risk

Reply:
I agree with adding the disclaimer.
What other modes besides development mode are available?